hist/histpainter/src/THistPainter.cxx

```diff
@@ -3657,7 +3657,7 @@ char *THistPainter::GetObjectInfo(Int_t px, Int_t py) const
 {
 
    if (!gPad) return (char*)"";
-   static char info[200];
+   TString info;
```

Comment:
@couet Here `static char` was used to allow use return value outside of the function.
You remove `static` qualifier and call `Form()` at the end - which is also workaround.
I propose just to add TString fObjectInfo into declaration part and simply return fObjectInfo.Data().
This is most adequate solution in my mind.

Reply:
@linev yes you are right I wa snot very happy of this solution.

Reply:
@linev You mean adding a data member TString fObjectInfo in THistPainter.h ?

Reply:
> You mean adding a data member TString fObjectInfo in THistPainter.h ?

Yes

Reply:
@linev: As GetObjectInfo is comments we will have very likely to declare it: 

mutable TString fObjectInfo

Reply:
yes, it makes sense